### PR TITLE
Ets 2190 top offer picture

### PR DESCRIPTION
**Pull request by BirgitPohl**
## PR to Top offer List

- Creates a picture atom 3:2 which can be used to put into a parent element
- It's using the image atom, which create a normal img- and not the picture-tag
- Moves images into its own folder for structural reasons

### CODE REVIEW

File: source/_patterns/00-atoms/12-images/03-image-ratio-3-2.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: Image 3:2
+state: undefined
```
Comment (susannegernandt) [inline source/_patterns/00-atoms/12-images/03-image-ratio-3-2.md:3]: Why undefined? Cause we still expand it?

File: source/_patterns/04-pages/03-teaser.mustache [hunk 2]
```diff
@@ -27,7 +27,7 @@
   <div class="teaser-list">
     <div class="teaser teaser-50 teaser-offer">
       <a href="#" class="teaser-box" role="button">
-        {{> molecules-teaser-images }}
+        {{> molecules-top-offer-image(imgSrc: '../../images/3-2-image-placeholder.png') }}
```
Comment (susannegernandt) [inline source/_patterns/04-pages/03-teaser.mustache:30]: Here we handover an image-source ...

File: source/_patterns/04-pages/03-teaser.mustache [hunk 3]
```diff
@@ -42,9 +42,9 @@
 
     <div class="teaser teaser-50 teaser-offer">
       <a href="#" class="teaser-box" role="button">
-        {{> molecules-teaser-images }}
+        {{> molecules-top-offer-image }}
```
Comment (susannegernandt) [inline source/_patterns/04-pages/03-teaser.mustache:45]: ... and here not. Correct?

File: source/scss/components/top-offer-list/_top-offer-list.scss [hunk 4]
```diff
@@ -23,3 +23,29 @@
...
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+
+  ::after {
+    content: "";
+    display: block;
+    padding-top: 66.66%;
+  }
+
```
Comment (susannegernandt) [inline source/scss/components/top-offer-list/_top-offer-list.scss:38]: Please talk to Denise again. If I got her right, the image should be centered centered. The cropping is OK the way, I did it and is maybe changed later.

File: source/_patterns/00-atoms/12-images/03-image-ratio-3-2.md (hunk 1 above)
Comment (BirgitPohl) [reply source/_patterns/00-atoms/12-images/03-image-ratio-3-2.md:3]: Solved via call. :)

File: source/_patterns/04-pages/03-teaser.mustache (hunk 2 above)
Comment (BirgitPohl) [reply source/_patterns/04-pages/03-teaser.mustache:30]: solved via call

File: source/_patterns/04-pages/03-teaser.mustache (hunk 3 above)
Comment (BirgitPohl) [reply source/_patterns/04-pages/03-teaser.mustache:45]: solved via call

File: source/scss/components/top-offer-list/_top-offer-list.scss (hunk 4 above)
Comment (BirgitPohl) [reply source/scss/components/top-offer-list/_top-offer-list.scss:38]: top to -9999 and bottom to -9999

File: source/_patterns/00-atoms/12-images/03-image-ratio-3-2.md (hunk 1 above)
Comment (filipe-esposito-fti) [reply source/_patterns/00-atoms/12-images/03-image-ratio-3-2.md:3]: What was the answer during the call?

File: source/_patterns/04-pages/03-teaser.mustache (hunk 2 above)
Comment (filipe-esposito-fti) [reply source/_patterns/04-pages/03-teaser.mustache:30]: What was the answer during the call?

File: source/_patterns/04-pages/03-teaser.mustache (hunk 3 above)
Comment (filipe-esposito-fti) [reply source/_patterns/04-pages/03-teaser.mustache:45]: What was the answer during the call?

Comment (susannegernandt) [reply source/_patterns/04-pages/03-teaser.mustache:45]: It gets a default value, if it's not handed over :)

File: source/_patterns/00-atoms/12-images/03-image-ratio-3-2.md (hunk 1 above)
Comment (BirgitPohl) [reply source/_patterns/00-atoms/12-images/03-image-ratio-3-2.md:3]: Undefined is a pattern lab state that we have defined to mark what elements have never been defined by designers.
And I felt, we needed this component.

**Review by susannegernandt — APPROVED** [review]

File: source/_patterns/04-pages/03-teaser.mustache (hunk 3 above)
Comment (BirgitPohl) [reply source/_patterns/04-pages/03-teaser.mustache:45]:
```
{{> molecules-top-offer-image(imgSrc: '../../images/3-2-image-placeholder.png') }}
```
Will define an individual imgSrc

If it is not defined:
```
{{> molecules-top-offer-image }}
```
Then there will be a default. And in this case, the default is defined in data.json.

**Review by filipe-esposito-fti — APPROVED** [review]

**Pull request merged**